zetaclient/zetacore/client_monitor.go

```diff
@@ -17,8 +17,9 @@ import (
 
 // constants for monitoring tx results
 const (
-	monitorInterval   = constant.ZetaBlockTime
-	monitorRetryCount = 10
+	defaultMonitorTimeout = 2 * time.Minute
```

Comment:
Is the default Timeout only used for the 7M vote? if so, it might be good to mention that in the comment